File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int l = nums.length;
        int x = 2*l;

        int[] ans = new int[x];
        for(int i=0;i<l;i++){
            ans[i]=nums[i];
        }
        int j=0;

        for(int i=l;i<x;i++){
            
            ans[i]=nums[j];
            j++;
        }

        return ans;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = {}
        for i in nums:
            freq[i]=freq.get(i,0)+1
        
        result =[]
        sorted_elements = sorted(freq, key=lambda x: freq[x], reverse=True)

        for n in range(k):
            if n>= len(sorted_elements):
                break
            elements= sorted_elements[n]
            result.append(elements)
        
        return result
